sdks/python/.pylintrc

```diff
@@ -119,10 +119,8 @@ disable =
   no-else-raise,
   no-else-return,
   no-member,
-  no-name-in-module,
```

Comment:
The warnings here could be related to the 'import-error' and 'no-member' errors. Also not sure how to fix them.